README.adoc

```diff
@@ -11,12 +11,13 @@ projects: []
 :toc:
 :icons: font
 :source-highlighter: prettify
-:project_id: draft-gs-template
+:project_id: gs-spring-cloud-gcp
 This guide walks you through the process of creating a Spring application.
```

Comment:
I'm not sure if the title I see at the top "The template for new guides and also the place to request them" is just an artifact of staging, or you have to change it somehow in the source.